counter/styles.css

```
@@ -1,29 +1,83 @@
+* { 
+  padding: 0;
+  margin: 0;
+}
```

Хорошая практика: всем элементам (и псевдо в том числе) всегда устанавливать border-box. Вот статья о боксовой модели -- это одна из главных вещей, которые нужно понимать в CSS: https://piccalil.li/blog/the-box-model-and-box-sizing/

```suggestion
*,
*::before,
*::after { 
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}
```